jrnl/plugins/__init__.py

```diff
@@ -16,6 +17,7 @@
 __exporters = [
     JSONExporter,
     MarkdownExporter,
+    PelicanMarkdownExporter,
```

Comment:
Since Markdown front matter is used by a lot more stuff than just Pelican, I think we should name it markdown-front-matter (and thus name this class MarkdownFrontMatterExporter).

Reply:
I tired naming it generically last time, and the exporter got dumped. I would like a specific name so it is clear exactly what the target use is.

Is there some YAML + Markdown standard is use outside of Pelican?

Reply:
> Is there some YAML + Markdown standard is use outside of Pelican?

There's no official standard, though it is widely used, and it is consistently referred to as "Front Matter" by a variety of well-known tools, including Jekyll, Hugo, and Assemble.

Reply:
> There's no official standard, though it is widely used, and it is consistently referred to as "Front Matter" by a variety of well-known tools, including Jekyll, Hugo, and Assemble.

Perhaps the concept is being well used, but each of static site generators have their own take on it, differing in deliminator as well as what keys they use. Pelican is the first sample below; the rest are taken from the documentation of the others you mention:

```
Title: My super title
Date: 2010-12-03 10:20
Modified: 2010-12-05 19:30
Category: Python
Tags: pelican, publishing
Slug: my-super-post
Authors: Alexis Metaireau, Conan Doyle
Summary: Short version for index and feeds

This is the content of my super blog post.
```

```
---
layout: post
title: Blogging Like a Hacker
---
```

```
---
title: "My First Post"
date: 2019-03-26T08:47:11+01:00
draft: true
---
```

```
---
page:
  title: Home
content: ../path/to/content.md
---
```

One blogger even went as far write a [conversion script](https://dev.to/wayofnumbers/how-i-migrate-my-data-science-blog-from-pelican-to-hugo-g63) when he moved from Pelican to Hugo.

The tl,dr version seems to be that if there is a standard, Pelican isn't following it, and thus a generic name seems out of place.